README.md

```diff
@@ -41,6 +41,7 @@ You can also:
 -   Report issues on the [Github repository](https://github.com/cozy-labs/cozy-desktop/issues)
 -   Chat with us on the [IRC channel](https://webchat.freenode.net/?channels=cozycloud)
 -   Mention us on [Twitter](https://twitter.com/mycozycloud)
+-   Synchronization problems are better reported by the help form included in the application (so logs are automatically attached to the request).
```

Comment:
I would probably move it to the top of the list, e.g.:

- Report synchronization issues using the help form included in the application (so logs are automatically attached to your message).
- Report other issues on the [GitHub repository](https://github.com/cozy-labs/cozy-desktop/issues)
- ...